docs/logging.md

```diff
@@ -0,0 +1,151 @@
+    ```bash
+    cd vhive
+    ```
+3. Create a directory for vHive logs:
+    ```bash
+    mkdir -p /tmp/vhive-logs
+    ```
+3. Run the node setup script:
+    ```bash
+    ./scripts/cloudlab/setup_node.sh > >(tee -a /tmp/vhive-logs/setup_node.stdout) 2> >(tee -a /tmp/vhive-logs/setup_node.stderr >&2)
+    ```
+    > **BEWARE:**
+    >
+    > This script can print `Command failed` when creating the devmapper at the end. This can be safely ignored.
+
+### 2. Setup Worker Nodes
+**On each worker node**, execute the following instructions **as a non-root user with sudo rights** using **bash**:
+1. Run the script that setups kubelet:
+    ```bash
+    ./scripts/cluster/setup_worker_kubelet.sh > >(tee -a /tmp/vhive-logs/setup_worker_kubelet.stdout) 2> >(tee -a /tmp/vhive-logs/setup_worker_kubelet.stderr >&2)
+    ```
+2. Start `containerd`:
+    ```bash
+    sudo containerd 2>&1 | tee /tmp/vhive-logs/containerd_log.txt
+    ```
+    
+    > **Note:**
+    >
+    > The logs from `containerd` will be generated at this path `/tmp/vhive-logs/containerd_log.txt`. You can change the log's file name or directory. The same applies for all the log files used in the following commands.
+    
+3. Open a new tmux pane with <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>"</kbd> or <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>%</kbd> then start `firecracker-containerd`:
+     > **Note:**
+    > You use <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>"</kbd> or <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>%</kbd> to split the current `tmux` pane horizontally or vertically. Additionally you can use <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>→</kbd>, <kbd>←</kbd>, <kbd>↑</kbd> or <kbd>↓</kbd> to navigate between panes.
+
+    ```bash
+    sudo PATH=$PATH /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 2>&1 | tee /tmp/vhive-logs/firecracker-containerd_log.txt
+    ```
+    
+
+4. Build vHive host orchestrator:
+    ```bash
+    source /etc/profile && go build
+    ```
+5. Open a new tmux pane with the same command as the previous step and start `vHive`:
+    ```bash
+    sudo ./vhive -dbg 2>&1 | tee /tmp/vhive-logs/vhive_log.txt
+    ```
+    > **Note:**
+    >
+    > By default, the microVMs are booted and vHive is started in debug mode enabled by `-dbg` flag. Additionally, you can enable snapshots after the 2nd invocation of each function by adding the `-snapshots` flag.
+    >
+    > If `-snapshots` and `-upf` are specified, the snapshots are accelerated with the Record-and-Prefetch (REAP) technique that we described in our ASPLOS'21 paper ([extended abstract][ext-abstract], [full paper](papers/REAP_ASPLOS21.pdf)).
+
+### 3. Configure Master Node
+**On the master node**, execute the following instructions below **as a non-root user with sudo rights** using **bash**:
+1. Start `containerd`:
+    ```bash
+    sudo containerd 2>&1 | tee /tmp/vhive-logs/containerd_log.txt
+    ```
+2. Split the `tmux` pane and run the script that creates the multinode cluster:
```

Comment:
You can create multiple tmux sessions, not necessarily split. Pick whatever is easier for the user so that just copy-paste of the commands work.